android/MobiledgeXSDKDemo/computervision/src/main/java/com/mobiledgex/computervision/PoseProcessorFragment.java

```diff
@@ -52,7 +52,7 @@
     private TextView mStdFull;
     private TextView mStdNet;
 
-    public static final String DEF_OPENPOSE_HOST_EDGE = "openpose.chicago-packet.mobiledgex.net";
+    public static final String DEF_OPENPOSE_HOST_EDGE = "posedetection.defaultedge.mobiledgex.net";
```

Comment:
Jim can change where this DNS entry points to, so we can change the default server without having to update the app.